src/server.rs

```diff
@@ -2,7 +2,8 @@ use std::collections::HashMap;
 use std::rc::Rc;
 use std::sync::{Arc, Mutex};
 
-use flux::semantic::walk;
+use flux::semantic::nodes::FunctionExpr;
+use flux::semantic::walk::{self, Node};
```

Comment:
Additional comment: just `use flux::semantic::walk` and then reference `Node` by `walk::Node`. There are too many things called `Node`, and I'd rather not scroll up to the import to figure out which one it is. At some point, maybe we'll be able to address this at the `libflux` level, but until then, it will at least make our intentions clear.